humans/static/javascripts/update_user_form.js

```diff
@@ -0,0 +1,47 @@
+$(".form__block label").addClass("smalltext");
+$(".form__block input[type=text]").addClass("text padding-settings");
+
+$("#id_keyserver_url").keyup(function(){
+  if($("#id_keyserver_url").val().length == 0) {
+    $("#id_public_key").attr("disabled",false);
+  } else {
+    $("#id_public_key").attr("disabled",true);
+  }
+});
+
+$("#id_keyserver_url").on("paste",function(){
+  if($("#id_keyserver_url").val().length == 0) {
+    $("#id_public_key").attr("disabled",false);
+  } else {
+    $("#id_public_key").attr("disabled",true);
+  }
+});
+
+$("#id_public_key").keyup(function(){
+  if($("#id_public_key").val().length == 0) {
+    $("#id_keyserver_url").attr("disabled",false);
+  } else {
+    $("#id_keyserver_url").attr("disabled",true);
+  }
+});
+
+$("#id_public_key").on("paste",function(){
+  if($("#id_public_key").val().length == 0) {
+    $("#id_keyserver_url").attr("disabled",false);
+  } else {
+    $("#id_keyserver_url").attr("disabled",true);
+  }
+});
+
+$(document).ready(function(){
+  if($("#id_keyserver_url").val().length != 0) {
+    $("#id_public_key").attr("disabled",true);
+  }
+  else if($("#id_public_key").val().length != 0) {
+    $("#id_keyserver_url").attr("disabled",true);
+  }
+});
+
+$("#id_server_signed").change(function() {
+  $('.server-signed-info-js .popuptext').toggleClass('show');
+});
```

Comment:
Instead of the `paste` and `keyup` events (4 listeners), why no use the `input``event just as said [here](http://stackoverflow.com/a/6458946/2729584). 

Also, perhaps would be better to enclose all this code in the scope of the "ready` call back. no?